lisa/node.py

```diff
@@ -344,6 +368,31 @@ def close(self) -> None:
         for node in self._list:
             node.close()
 
+    def from_either(
```

Comment:
If they are merged, the name can be from_runbook.

Reply:
But all of them are from runbook (`from_requirement` included).